src/main/java/org/openlmis/buq/web/buq/BottomUpQuantificationController.java

```diff
@@ -204,4 +204,22 @@ public ResponseEntity<String> getBottomUpQuantificationAuditLog(@PathVariable("i
         page);
   }
 
+  /**
+   * Endpoint to submit a bottomUpQuantification.
+   *
+   * @param id UUID of bottom-up quantification which we want to update.
+   * @param bottomUpQuantificationDto A bottom-up quantification DTO bound to the request body.
+   * @return updated bottom-up quantification dto.
+   */
+  @PutMapping("/{id}/submit")
+  @ResponseStatus(HttpStatus.OK)
+  @ResponseBody
+  public ResponseEntity<BottomUpQuantificationDto> submitBottomUpQuantification(
```

Comment:
I think for code consistency we should stay with returning DTO, without ResponseEntity, if we don't need to pass any extra data about response.

Reply:
I've changed the method to return DTO.